### ISSUE TITLE
Reportback: Tests for CRUD functions

### ISSUE THREAD

**Issue by aaronschachter** [DoSomethingArchive/legacy-website#456]
Depends on #455 

Web tests for:
- [x] dosomething_reportback_schema
- [ ] dosomething_reportback_insert_reportback
- [ ] dosomething_reportback_update_reportback
- [x] dosomething_reportback_save
- [x] dosomething_reportback_set_properties
- [x] dosomething_reportback_set_files
- [ ] dosomething_reportback_log_insert
- [ ] dosomething_reportback_entity_insert
- [ ] dosomething_reportback_entity_update
- [ ] dosomething_reportback_entity_delete
- [ ] dosomething_reportback_exists
- [ ] dosomething_reportback_access
- [ ] dosomething_reportback_form

**Comment by barryclark** [comment]
@aaronschachter Still for Tues?

**Comment by aaronschachter** [comment]
This would be a nice to have, but I'd imagine this is not critical for launch.  Was hoping I'd have bandwidth but not sure about that yet.

**Comment by barryclark** [comment]
Cool, will leave it in 0.2.0 for now.

**Comment by barryclark** [comment]
Backlogging this as it's not critical for Tuesday launch.
